### ISSUE TITLE
Add alt text to 23+ images missing it

### ISSUE THREAD

**Issue by Sachinchaurasiya360** [Sachinchaurasiya360/InternHack#23]
- AdminBadgesPage.tsx, BlogPostPage.tsx, NotFoundPage.tsx, TalentCard.tsx

**Comment by thisisAtharv** [comment]
Hi @Sachinchaurasiya360 , I would love to take this on! Could you please assign it to me?

**Comment by Sachinchaurasiya360** [comment]
done
